apps/browser/qml/pages/components/BookmarkEditDialog.qml

```diff
@@ -83,20 +83,34 @@ UserPromptDialog {
                 id: urlField
                 text: url
 
-                acceptableInput: text.length > 0
                 onActiveFocusChanged: if (!activeFocus) errorHighlight = !acceptableInput
                 onAcceptableInputChanged: if (acceptableInput) errorHighlight = false
 
                 //: Label for textfield to edit bookmark's URL
                 //% "URL"
                 label: qsTrId("sailfish_browser-la-url_editor")
-                //% "URL is required"
-                description: errorHighlight ? qsTrId("sailfish_browser-la-url_editor_error") : ""
+                description: {
+                    if (!errorHighlight) {
+                        return ""
+                    } else if (text.length > 0) {
+                        var scheme = root.url.match(/^(https?):\/\/.+/)
+
+                        //% "URL scheme (%1) missing"
+                        return qsTrId("sailfish_browser-la-missing_url_scheme").arg(scheme.length > 1 ? scheme[1] : "https")
```

Comment:
If `root.url` doesn't match the regex (e.g. attempting to bookmark a local file), `scheme` will be null which breaks the `scheme.length` check, so I guess there should be a null check in here too.

Reply:
Fixed.